crates/scroll/revm/src/states/bundle.rs

```diff
@@ -72,21 +72,13 @@ impl From<(BundleState, &ScrollPostExecutionContext)> for ScrollBundleState {
 }
 
 // This conversion can cause a loss of information since performed without additional context.
-#[cfg(any(not(feature = "scroll"), feature = "test-utils"))]
+#[cfg(any(feature = "mpt", feature = "test-utils"))]
```

Comment:
this was requested by the zkvm team

Reply:
Is there any reason why you remove the `not(feature = "scroll")` gate here and keep it in the modification of `impl<DB: DatabaseRef> FinalizeExecution for State<CacheDB<DB>>`.

Reply:
the `not(feature = "scroll")` didn't make sense in the `reth-scroll-revm` crate, because the whole `states` module is gated with `feature = "scroll"` so I removed it.